hack/check-gh-chart-versions.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
```

Comment:
```suggestion
set -eo pipefail
```

Sets the shell options to exit immediately if any command exits with a non-zero status, and to exit with a non-zero status if any command in a pipeline fails.
- set -e: Exit immediately if a command exits with a non-zero status.
- set -o pipefail: Exit with a non-zero status if any command in a pipeline fails.

Reply:
Also here, can we add some comments to describe what this script is / does and how it can be called?

```bash
# Checks the chart versions against an input version. Fails on mismatch.
#
# Usage:
#   check-gh-chart-versions.sh <VERSION>
```